misago/users/management/commands/removeips.py

```diff
@@ -0,0 +1,14 @@
+
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand
+
+UserModel = get_user_model()
+
+class Command(BaseCommand):
+    help = "Deletes all user registered IPs"
```

Comment:
This should be "Anonymizes users IPs stored for longer than set in MISAGO_IP_STORE_TIME."